<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>531</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="createEventTab">
       <attribute name="title">
        <string>Event Creation</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCalendarWidget" name="calendarWidget"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLineEdit" name="lineTitle">
              <property name="text">
               <string>Event Title</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextDesc">
              <property name="plainText">
               <string>Event Description</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="dateTimeStart">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="dateTimeEnd">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="createButton">
              <property name="text">
               <string>Create</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Employee">
       <attribute name="title">
        <string>Employees</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="FindEmployee">
            <item>
             <widget class="QLabel" name="Namelabel">
              <property name="text">
               <string>Employee Name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Name"/>
            </item>
            <item>
             <widget class="QLabel" name="IDLabel">
              <property name="text">
               <string>Employee ID Num:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="ID">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhNone</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="EmployeeInfo" stretch="0,0,0,0">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="emailLabel">
              <property name="text">
               <string>Email:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="email"/>
            </item>
            <item>
             <widget class="QLabel" name="HrsReqLabel">
              <property name="text">
               <string>Weekly Hours Requested:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="HrsReq"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="Monday">
            <item>
             <widget class="QLabel" name="MonLabel">
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Monday:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="MonStart">
              <property name="displayFormat">
               <string>h AP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="MonEnd">
              <property name="displayFormat">
               <string>h AP</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="Tuesday">
            <item>
             <widget class="QLabel" name="TueLabel">
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Tuesday:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="TueStart">
              <property name="displayFormat">
               <string>h AP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="TueEnd">
              <property name="displayFormat">
               <string>h AP</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="Wedsday">
            <item>
             <widget class="QLabel" name="WedLabel">
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Wedsday:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="WedStart">
              <property name="displayFormat">
               <string>h AP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="WedEnd">
              <property name="displayFormat">
               <string>h AP</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="Thursday">
            <item>
             <widget class="QLabel" name="ThurLabel">
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Thursday:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="ThurStart">
              <property name="displayFormat">
               <string>h AP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="ThurEnd">
              <property name="displayFormat">
               <string>h AP</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="Fri">
            <item>
             <widget class="QLabel" name="FriLabel">
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Friday:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="FriStart">
              <property name="displayFormat">
               <string>h AP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="FriEnd">
              <property name="displayFormat">
               <string>h AP</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="Saturday">
            <item>
             <widget class="QLabel" name="SatLabel">
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Saturday:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="SatStart">
              <property name="displayFormat">
               <string>h AP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="SatEnd">
              <property name="displayFormat">
               <string>h AP</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="Sunday">
            <item>
             <widget class="QLabel" name="SunLabel">
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Sunday:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="SunStart">
              <property name="displayFormat">
               <string>h AP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="SunEnd">
              <property name="currentSection">
               <enum>QDateTimeEdit::HourSection</enum>
              </property>
              <property name="displayFormat">
               <string>h AP</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="UseBox">
            <item>
             <widget class="QPushButton" name="SaveButton">
              <property name="minimumSize">
               <size>
                <width>850</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox"/>
            </item>
            <item>
             <widget class="QPushButton" name="Find">
              <property name="text">
               <string>Find Empolyee With ID</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="EmployeeList">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <attribute name="title">
        <string>Employee List</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QListView" name="listView"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>Create Employee</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Delete Employee</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_2"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="WeeklyShifts">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Weekly Shifts</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="SaveShifts">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="Days">
              <item>
               <widget class="QLabel" name="ShiftDaysLabel">
                <property name="text">
                 <string>Day:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="ShiftDays"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="StartTime">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Start Time:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTimeEdit" name="StartTimeShift">
                <property name="displayFormat">
                 <string>h AP</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="EndTime">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>End Time:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTimeEdit" name="EndTimeShift"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="EmployeesNeeded">
              <item>
               <widget class="QLabel" name="Employees">
                <property name="text">
                 <string>Employees In Shift:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="EmployeesInShift"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Delete Shift</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="CreateNewShift">
              <property name="text">
               <string>Create New Shift</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Sunday:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Monday:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Tuesday:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Wedsday:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Thursday:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Friday:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Saturday:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QListView" name="MondayShifts"/>
            </item>
            <item>
             <widget class="QListView" name="TuesdayShifts"/>
            </item>
            <item>
             <widget class="QListView" name="WedsdayShifts"/>
            </item>
            <item>
             <widget class="QListView" name="ThursdayShifts"/>
            </item>
            <item>
             <widget class="QListView" name="FridayShifts"/>
            </item>
            <item>
             <widget class="QListView" name="SaturdayShifts"/>
            </item>
            <item>
             <widget class="QListView" name="SundayShifts"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Page</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
